Core/src/main/java/com/plotsquared/core/command/ListCmd.java

```diff
@@ -120,25 +120,22 @@ private String[] getArgumentList(PlotPlayer<?> player) {
         return args.toArray(new String[args.size()]);
     }
 
-    public void noArgs(PlotPlayer<?> player) {
-        player.sendMessage(
-                TranslatableCaption.of("commandconfig.subcommand_set_options_header"),
-                Templates.of("values", Arrays.toString(getArgumentList(player)))
-        );
-    }
-
     @Override
     public boolean onCommand(PlotPlayer<?> player, String[] args) {
+        // finalArgs is reset to "mine" in the event that a user runs the command /plot list with no args,
+        // otherwise the original arguments will be used.
+        String[] finalArgs;
         if (args.length < 1) {
-            noArgs(player);
-            return false;
+            finalArgs = new String[] {"mine"};
```

Comment:
I'm not sure there's a need to create a new variable since we're effectively just overwriting the args var here anyway, and there's no need for args to be "final" (and they're not actually final)